core/src/test/java/io/aiven/klaw/service/TopicControllerServiceTest.java

```diff
@@ -382,74 +685,219 @@ public void getAllTopics() {
   }
 
   @Test
-  @Order(16)
-  public void getTopicsSuccess1() throws Exception {
+  @Order(29)
+  public void getAllTopics2() {
```

Comment:
We should give it a proper name instead of `1` and `2`.

Reply:
Agree, updated.